### PR TITLE
Assign PRs created by the bot to the repo admin

### PR DESCRIPTION
**Pull request by jleach**
When creating a PR it now gets assigned to the repo admins. This was done to better ensure ownership of the issues.

### CODE REVIEW

File: __tests__/ghutils.test.ts [hunk 1]
```diff
@@ -39,58 +35,24 @@ const configResponse = JSON.parse(fs.readFileSync(p2, 'utf8'));
...
+const p5 = path.join(__dirname, 'fixtures/repo-get-pulls.json');
+const listPulls = JSON.parse(fs.readFileSync(p5, 'utf8'));
 
 const p6 = path.join(__dirname, 'fixtures/repo-list-commits.json');
 const listCommits = JSON.parse(fs.readFileSync(p6, 'utf8'));
 
 const p7 = path.join(__dirname, 'fixtures/org-user-ismember.json');
 const memberhip = JSON.parse(fs.readFileSync(p7, 'utf8'));
 
+const p8 = path.join(__dirname, 'fixtures/repo-get-collaborators.json');
```
Comment (patricksimonian) [inline __tests__/ghutils.test.ts]: did you know you should be able to directly import `json` files

`import p8 from '../fixtures/foo.json'` 

you may need to update babel settings to ensure you don't transform `.json` files

File: __tests__/src/helper.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    .reply(200, {});
+
+let app;
+let github;
+
+app = new Application();
+app.app = { getSignedJsonWebToken: () => 'xxx' };
+app.load(robot);
+
+github = {
```
Comment (patricksimonian) [inline __tests__/src/helper.ts]: is this a mock?

Comment (jleach) [reply __tests__/src/helper.ts]: Yup.

File: __tests__/ghutils.test.ts (hunk 1 above)
Comment (jleach) [reply __tests__/ghutils.test.ts]: Thanks for the protip. Will refactor time permitting.

**Pull request merged**